pkgs/applications/misc/twitch-chat-downloader/default.nix

```
@@ -20,6 +20,11 @@ buildPythonApplication rec {
     hash = "sha256-mV60ygrtQa9ZkJ2CImhAV59ckCJ7vJSA9cWkYE2xo1M=";
   };
 
+  postPatch = ''
+    substituteInPlace tcd/example.settings.json --replace-fail \
+    jzkbprff40iqj646a697cyrvl0zt2m6 kd1unb4b3q4t58fwlpcbzcbnm76a8fp
+  '';
```

Could you link https://github.com/TheDrHax/Twitch-Chat-Downloader/pull/16 in a comment for reference as to why this change is being made?

```suggestion
  postPatch = ''
    # Update client ID for Twitch changes
    # See: <https://github.com/TheDrHax/Twitch-Chat-Downloader/pull/16>
    substituteInPlace tcd/example.settings.json \
      --replace-fail jzkbprff40iqj646a697cyrvl0zt2m6 kd1unb4b3q4t58fwlpcbzcbnm76a8fp
  '';
```